boards/BMS/Inc/Io/Io_CellVoltages.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdlib.h>
+#include "App_SharedExitCode.h"
+
+/**
+ * Read cell voltages (100µV) from the cell monitoring chips.
+ * @return EXIT_CODE_OK if cell voltages were acquired successfully from all
+ * cell monitoring chips. Else, EXIT_CODE_ERROR.
+ */
+ExitCode Io_CellVoltages_ReadCellVoltages(void);
+
+/**
+ * Get the pointer to the 2D array containing converted cell voltages (100µV)
```

Comment:
Similar comment about `(100uv)` here, I think it makes sense to only mention the units in the `@return`.

Reply:
fixed